internal/kafka/test/integration/admin_kafka_test.go

```diff
@@ -1146,6 +1155,148 @@ func TestAdminKafka_Update(t *testing.T) {
 				g.Expect(result.MaxDataRetentionSize.Bytes).To(gomega.Equal(dataRetentionSizeBytes))
 			},
 		},
+		{
+			name: "should have no effect on kafka status when setting suspended to false on unsuspended kafka",
+			args: args{
+				ctx: func(h *coreTest.Helper) context.Context {
+					return NewAuthenticatedContextForAdminEndpoints(h, []string{testFullRole})
+				},
+				kafkaID: sampleKafkaID1,
+				kafkaUpdateRequest: adminprivate.KafkaUpdateRequest{
+					Suspended: &falseB,
+				},
+			},
+			verifyResponse: func(result adminprivate.Kafka, resp *http.Response, err error) {
+				g.Expect(err).To(gomega.BeNil())
```

Comment:
```suggestion
				g.Expect(err).NotTo(gomega.HaveOccurred())
```

Reply:
cc @pawelpaszki, this change is from https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1344